_data/events.yml

```diff
@@ -79,28 +79,38 @@
   twitter: "https://x.com/itenium_be/status/1859552439352910029"
   facebook: "https://www.facebook.com/share/p/14MyZEf6yo/"
 
-- name: "DDD & BDD"
-  by: "Gaëtan"
-  subTitle: ""
+- name: "Soft Skill Sessie : presentatietechnieken en spreke"
```

Comment:
The dutch name is incomplete
The name & the subTitle are also exactly the same (one in dutch, the other in english)

Maybe just put "Soft Skill Sessie" as the name